lms/static/sass/base/_variables.scss

```diff
@@ -166,7 +166,7 @@ $m-gray-l1: rgb(151,153,155); // #97999B
 $m-gray-l2: rgb(164,166,168); // #A4A6A8
 $m-gray-l3: rgb(177,178,180); // #B1B2B4
 $m-gray-l4: rgb(245,245,245); // #F5F5F5
-$m-gray-d1: rgb(125,127,131); // #7D7F83
+$m-gray-d1: rgb(111, 112, 116); // #6f7074 AA compliant
```

Comment:
Thanks for the change. 

Since all of our color combinations should be AA/AAA compliant, do we need the comment here in our variable definitions? To me, the comment may not make sense without context of when that is compliant (i.e. what background-color is this compliant against?). 

My recommendation is to just remove the "AA Compliant" bit to avoid confusion.